convert_sc_client_lookup.R

```diff
@@ -0,0 +1,271 @@
+#####################################################
+# Social Care Lookup
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description -
+#####################################################
+
+## packages ##
+library(dplyr)
+library(dbplyr)
+library(tidyr)
+
+
+
+
+## data ##
+
+######################################################
+# set-up conection to platform
+db_connection <- odbc::dbConnect(
+  odbc::odbc(),
+  dsn = "DVPROD",
+  uid = Sys.getenv("USER"),
+  pwd = rstudioapi::askForPassword("password")
+)
+###################################################
+
+# read in data - social care 2 client
+sc <- tbl(db_connection, in_schema("social_care_2", "client")) %>%
+  select(
+    sending_location,
+    social_care_id,
+    financial_year,
+    financial_quarter,
+    dementia,
+    mental_health_problems,
+    learning_disability,
+    physical_and_sensory_disability,
+    drugs,
+    alcohol,
+    palliative_care,
+    carer,
+    elderly_frail,
+    neurological_condition,
+    autism,
+    other_vulnerable_groups,
+    living_alone,
+    support_from_unpaid_carer,
+    social_worker,
+    type_of_housing,
+    meals,
+    day_care
+  ) %>%
+  collect() %>%
+  arrange(
+    sending_location,
+    social_care_id,
+    financial_year,
+    financial_quarter
+  )
+
+
+## year of interest ##
+year <- 2019
```

Comment:
I think it better to take the year as 1920 here, as that's what all other scripts use. Then convert for the filter.